install_config/configuring_pipeline_execution.adoc

```diff
@@ -7,26 +7,32 @@
 :experimental:
 :toc: macro
 :toc-title:
+:prewrap!:
 
 toc::[]
 
 
 == Overview
 
-{product-title} utilizes Jenkins to execute continuous development pipelines.  To
-provide a seamless experience for users, the first time a user creates a build
-configuration using the xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[pipeline]
-strategy type, a set of Jenkins related resources will be created within the user's project.
-By default, the system will look for a template named `*jenkins*` in the `*openshift*` namespace and
-instantiate it within the user's project.  {product-title} ships with a default template definition
-which will, upon instantiation, create the following in the user's project:
+{product-title} utilizes Jenkins to execute continuous development pipelines.
+To provide a seamless experience for users,
+the first time a user creates a build configuration using the
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline]
+strategy type,
+a set of Jenkins related resources will be created within the user's project.
+By default, the system will look for a template named `*jenkins*`
+in the `*openshift*` namespace and instantiate it within the user's project.
+{product-title} ships with a default template definition which will,
```

Comment:
first off, those are not the templates used to instantiate a jenkins server for pipeline buildconfigs, those are just quickstarts for standing up a one-off jenkins deployment.  

The jenkins template that we instantiate by default for pipelines is here:
https://github.com/openshift/origin/blob/master/examples/jenkins/pipeline/jenkinstemplate.json

I don't know where (if anywhere) it will be on the filesystem, that depends on the installer and i'm not sure how it's handling this file.

If you want to view the content of the template, assuming it was registered with the cluster at install time, you can run:
oc get template jenkins -n openshift -o yaml

and that template currently uses ephemeral storage, though that's liable to/subject to change (and there is an expectation admins will tweak the "default" template for their cluster)